packages/apostrophe-astro/components/AposArea.astro

```diff
@@ -67,10 +67,9 @@ function getWidgetOptions(options) {
         ...item._options,
         ...widgetOptions[item.type],
       };
-      const { _options, ...cleanItem } = item;
       return (
         <WidgetComponent
-          widget={cleanItem}
+          widget={item}
```

Comment:
widgets need to keep their options in order to be able to call `stylesElements` and `stylesAttributes` with `widget`:

```astro
// in a project, when setting stylesWrapper to false:

import { stylesElements, stylesAttributes } from '@apostrophecms/apostrophe-astro/lib/aposStyles.js';
const { widget } = Astro.props;
---
<Fragment set:html={ stylesElements(widget) } />
<article {...stylesAttributes(widget, { class: 'fancy-article' }) }>
  more of their own content here
</article>
```